src/content/en/2020/seo.md

```diff
@@ -23,3 +23,835 @@ queries: 07_SEO
 #featured_stat_label_3: TODO
```

Comment:
@aleyda can you think of a nice quote and three featured stats to entice people into the chapter. These will appear randomly on the home page.

Reply:
Hi @bazzadp, thanks for asking :) 

@ipullrank @fellowhuman1101 - what do you think to include for: 

- **The main quote:** "The link discoverability for major JS frameworks used for SPAs increased dramatically compared to 2019. By testing mobile navigation links for hashed URLs, we saw -53%  instances of uncrawlable links from sites using React, -58% fewer from Vue.js powered sites, and a -91% reduction from Angular SPAs."

- **The 3 featured stats:** 

1) 10.84% of mobile home pages and 16.18% of desktop ones are not including the viewport tag at all, suggesting that they are not yet mobile friendly.

2) 33.13% of websites scored 'Good' Core Web Vitals for desktop while only 19.96% of their mobile counterparts passed the Core Web Vitals assessment.  

3) The median mobile site displays 11.5% more words when rendered than its raw HTML, showing a reliance on client side JavaScript to show web pages content. 

Let me know what you think of these or if you would prefer to highlight other stats/findings! Thanks :)

Reply:
@aleyda I think the main quote talks about what's gotten better, but we should consider using our main quote as the opportunity to alert people to the big problem of their being a disparity between mobile and desktop.

Reply:
@ipullrank I see what you mean... especially due to the Mobile First Index. What about: 

"Despite the growing use of mobile devices and Google’s move to a Mobile first index, non-trivial disparities were found across mobile vs. desktop pages, like the one between mobile and desktop links, illustrated in the 62 links on mobile versus the 68 links on desktop for the median web page."? 

cc @fellowhuman1101

Reply:
@aleyda muy bien para mi

Reply:
@ipullrank gracias Mike! 

@fellowhuman1101 Is it good for you too Jamie? :)

Reply:
@aleyda Looks good!

Reply:
Thanks for the confirmation @fellowhuman1101 :) 

@bazzadp @rviscomi I just added the quote and stats and [submitted them here](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1628). I noticed there were "labels" for them but didn't know what should I add there. Could you please let me know to include them too if necessary? :) Thanks again

Reply:
Thanks everyone! I've followed up in the linked PR about the next steps to get this included. To summarize, let's apply any further changes directly to the open [PR](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1625) to keep everything in one place.

BTW @fellowhuman1101 there's a TODO in the PR  for you to add your bio, and I've also sent you an [invite](https://github.com/HTTPArchive) to join the Authors Team on GitHub, so we can assign issues/PRs to you.